### PR TITLE
PYIC-7334: Add F2F p1 journeys api-tests

### PR DESCRIPTION
**Pull request by AmritSidhu**
### What changed

Add F2F p1 journeys api-tests

### Issue tracking

- [PYIC-7334](https://govukverify.atlassian.net/browse/PYIC-7334)

### Environment variables or secrets

- [x] No environment variables or secrets were added or changed

[PYIC-7334]: https://govukverify.atlassian.net/browse/PYIC-7334?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: api-tests/features/p1-journeys.feature [hunk 1]
```diff
@@ -192,3 +192,58 @@ Feature: P1 journey
...
+    When I submit 'kenneth-current' details to the CRI stub
+    Then I get a 'fraud' CRI response
+    When I submit 'kenneth-score-1-history-0' details to the CRI stub
+    Then I get a 'f2f' CRI response
+    When I submit 'kenneth-passport-valid' details with attributes to the CRI stub
+      | Attribute          | Values                                      |
+      | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":2} |
+    Then I get a 'page-face-to-face-handoff' page response
+
+  Scenario: P1 F2F Low confidence journey should have relevent photoid option text
```
Comment (DanCorderIPV) [inline api-tests/features/p1-journeys.feature]: Is this test needed? We're not testing any text and the test steps are just a subset of `P1 F2F Journey` above

File: api-tests/features/p1-journeys.feature [hunk 2]
```diff
@@ -192,3 +192,58 @@ Feature: P1 journey
       | Attribute          | Values                                      |
       | evidence_requested | {"scoringPolicy":"gpg45","strengthScore":2} |
     Then I get a 'page-face-to-face-handoff' page response
+
+  Scenario: P1 F2F Support low strength F2F documents for thin file users
```
Comment (DanCorderIPV) [inline api-tests/features/p1-journeys.feature]: I realise that you've probably just copied this test name over from an end to end test, but there's no KBV step here so I'm not sure what `thin file` has to do with this test.

**Review by DanCorderIPV — CHANGES_REQUESTED** [review]

File: api-tests/features/p1-journeys.feature (hunk 2 above)
Comment (blakeyp) [reply api-tests/features/p1-journeys.feature]: I think this is referring to thin _fraud_ file - when a user has a low fraud/activity score and we need to pass a required strength to F2F CRI

Comment (blakeyp) [reply api-tests/features/p1-journeys.feature]: will be clearer if we just rename it "thin fraud file" users?

File: api-tests/features/p1-journeys.feature [hunk 3]
```diff
@@ -192,3 +192,58 @@ Feature: P1 journey
...
+  Scenario: P1 F2F Low confidence journey should have relevent photoid option text
+    Given I activate the 'p1Journeys' feature set
+    When I start a new 'low-confidence' journey
+    Then I get a 'page-ipv-identity-document-start' page response
+    When I submit an 'end' event
+    Then I get a 'prove-identity-no-photo-id' page response with context 'nino'
+    When I submit an 'end' event
+    Then I get a 'page-ipv-identity-postoffice-start' page response with context 'lastChoice'
+
+  Scenario: P1 F2F Support low strength F2F documents for P2 user
```
Comment (blakeyp) [inline api-tests/features/p1-journeys.feature]: This is the regression scenario for P2 but we might already have a pretty identical test to this in the P2 tests? if so I don't think we need this

**Review by blakeyp — APPROVED** [review]

**Review by DanCorderIPV — APPROVED** [review]

**Pull request merged**